### ISSUE TITLE
Shell menu in custom start menu folders don't update

### ISSUE THREAD

**Issue by TurboJens** [Open-Shell/Open-Shell-Menu#578]
**Describe the bug**
I have a folder for network connections. When right click on a connection and select Disable the network card is disabled, but next time the menu choice should change to "Enable" but it still says "Disable" and since it's already disabled nothing happens upon selection.

**To Reproduce**
1. Create new item on start menu, command "Network connections", display "Display as a menu"
2. Hover ocer the new folder, right-click on ex. 'Ethernet'
3. Select 'Disable'
4. Repeat step 2 and look for 'Disable' wich should have been 'Enable' item.

**Expected behavior**
When right-click on a disabled connection shell menu should show 'Enable' instead of 'Disable', preferably also the icon should be updated according to status.

**Screenshots**

**Version:**
 - Open-Shell: [e.g. 4.4.160]
 - OS: Windows 10 Pro 20H2 (same issue with previous versions)

**Additional context**
![Screenshot 2020-12-17 104220](https://user-images.githubusercontent.com/76071330/102470679-95d31000-4054-11eb-8fa4-fcac7617146a.png)

**Comment by bonzibudd** [comment]
I'm able to reproduce this.

However it appears that this is a Windows glitch, not Open-Shell. If I make a shortcut on the desktop, the same thing happens.
![image](https://user-images.githubusercontent.com/61938331/102493970-f573f000-4041-11eb-82df-6f2b572052d5.png)

**Comment by bonzibudd** [comment]
@TurboJens are you able to reproduce the same behavior with a desktop icon like I was able to? If so, I think that this is probably a limitation from Microsoft's part that creates problems in various situations, including Open-Shell.

**Comment by blackcrack** [comment]
yes, Open-shell have normally nothing to do with Network Connection, Drivers or displaying of networking things.

**Comment by TurboJens** [comment]
> 
> 
> @TurboJens are you able to reproduce the same behavior with a desktop icon like I was able to? If so, I think that this is probably a limitation from Microsoft's part that creates problems in various situations, including Open-Shell.

Hello, Yes, you are right. I got the excact same result. I did som goole search, and found that this har obvious been erratic from MS since Win10 at least. Seem to work fine in Windows 7.
